packages/frontend/hooks/src/use-workspace-info.ts

```diff
@@ -0,0 +1,30 @@
+import type { Workspace, WorkspaceMetadata } from '@affine/workspace';
+import { workspaceManagerAtom } from '@affine/workspace/atom';
+import { useAtomValue } from 'jotai';
+import { useEffect, useState } from 'react';
+
+export function useWorkspaceInfo(
+  meta: WorkspaceMetadata,
+  workspace?: Workspace
+) {
+  const workspaceManager = useAtomValue(workspaceManagerAtom);
+
+  const [information, setInformation] = useState<{
+    avatar?: string;
+    name?: string;
+  } | null>(null);
```

Comment:
it looks to me this hook may always return non-null values since `getInformation` will create one instance if not exist.
maybe init initial value using `workspaceManager.list.getInformation(meta).info`?

Reply:
yeah, the init value of useState could be a getter function